src/css/School.css

```diff
@@ -7,12 +7,49 @@
 */
 
 .school {
-    border: 3px solid #EC8556;
+    /* background-color: #FAE9CD; */
+    border: 5px solid #EC8556;
     border-radius: 5px;
     margin: 2%;
     padding: 1%;
     color: #154874;
     display: flex;
     flex-direction: column;
-    justify-content: flex-start;
+    /* align-items: flex-start; */
+}
+
+.school-header {
+    padding: 1%;
+    display: flex;
+    flex-direction: row;
+    border-bottom: 2px solid #67C9E2;
+    justify-content: space-between;
+}
+
+.school-name {
+    margin: 0;
+    padding: 0;
+    text-align: left;
+    width: 75%
+}
+
+.bookmark {
+    margin: 0;
+    padding: 0;
+    width: 15%;
+    font-size: 2em;
+}
+
+.url {
+    margin: 1%;
+    font-size: 2em;
+    color: #154874;
+}
+
+.detail {
+    text-align: left;
+}
+
+.bold {
+    font-weight: bold;
```

Comment:
Have you thought about refactoring to sass? Might cut down a few lines of code here.

Reply:
Agreed, thanks! Will consider this for a future branch.